test/test_client.py

```diff
@@ -504,6 +504,52 @@ def test_missing(self):
     self.client.status('foo')
 
 
+class TestChangeOwner(_TestSession):
+
+  def test_directory(self):
+    new_owner = 'newowner'
+    self.client._mkdirs('foo')
+    self.client.change_owner('foo', new_owner)
+    status = self.client.status('foo')
+    status = self.client.status('foo')
+    eq_(status['owner'], new_owner)
```

Comment:
In the case (though unlikely) where the original owner is `'newowner'`, this test won't do anything. Could you add a change to a second (different) user to cover all cases? (Same for all other tests.)